From 9493dc203ba49843e1cd594defa7c4999414fe66 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:32:02 -0400
Subject: [scsi] hpsa: remove bad unlikely annotation from device list updating code

Message-id: <1410186734-19986-72-git-send-email-thenzl@redhat.com>
Patchwork-id: 92092
O-Subject: [RHEL6.6 e-stor PATCH 71/83] hpsa: remove bad unlikely annotation from device list updating code
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Reviewed-by: Justin Lindley <justin.lindley@hp.com>
Reviewed-by: Mike Miller <michael.miller@canonical.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 8645291b8d1bac28efb803e768e91f24c814679c)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index 57e699b..906e882 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -3073,7 +3073,7 @@ static void hpsa_update_scsi_devices(struct ctlr_info *h, int hostno)
   ndev_allocated++;
  }
 
- if (unlikely(is_scsi_rev_5(h)))
+ if (is_scsi_rev_5(h))
   raid_ctlr_position = 0;
  else
   raid_ctlr_position = nphysicals + nlogicals;
-- 
1.7.1